File: python/get_consents.py
from client import create_native_client
import nativeconnect.api.v1.nativeconnect_pb2 as native


def main():
    client = create_native_client()

    request = native.GetConsentsRequest()

    response = client.GetConsents(request)

    print(response)


if __name__ == "__main__":
    main()

File: typescript/src/get_consents.ts
import { createNodeClient } from "./client"
import { create, toJsonString } from "@bufbuild/protobuf";
import {
  GetConsentsRequestSchema,
  GetConsentsResponseSchema
} from "@buf/nativeconnect_api.bufbuild_es/nativeconnect/api/v1/nativeconnect_pb";

const client = createNodeClient()

const request = create(GetConsentsRequestSchema)

const response = await client.getConsents(request)

console.log(toJsonString(GetConsentsResponseSchema, response, { prettySpaces: 2 }))
